.azure/azure-pipelines.ci.yml

```
@@ -406,7 +406,7 @@ stages:
       platform: windows
       tls: schannel
       logProfile: Full.Light
-      extraArgs: -Kernel
+      extraArgs: -Kernel -Filter -Handshake/WithHandshakeArgs6.ConnectClientCertificate/*
```

Try this (same for above):
```suggestion
      extraArgs: -Kernel -Filter -*Handshake/WithHandshakeArgs6.ConnectClientCertificate/*
```